### PR TITLE
Terminate file_import tasks upon cancellation of an Analysis

### PR DESCRIPTION
**Pull request by scottx611x**
Addresses #1730

### CODE REVIEW

File: refinery/core/models.py [hunk 1]
```diff
@@ -1740,6 +1744,26 @@ def attach_outputs_downloads(self):
                     "No file found for '%s' in download '%s' ('%s')",
                     analysis_result.file_store_uuid, self.name, self.uuid)
 
+    def terminate_file_import_tasks(self):
+        """
+        Gathers all UUIDs of FileStoreItems used as inputs for the Analysis,
+        and trys to terminate their import_file tasks if possible
```
Comment (jkmarx) [inline refinery/core/models.py]: Gathers all UUIDs of FileStoreItems used as inputs for the Analysis, and try to terminate their import_file tasks

Comment (scottx611x) [inline refinery/core/models.py]: @jkmarx  ???

Comment (jkmarx) [inline refinery/core/models.py]: suggested comment revision

Comment (jkmarx) [inline refinery/core/models.py]: approved regardless

Comment (scottx611x) [inline refinery/core/models.py]: Oh sorry, my eyes aren't awake yet, I thought it was the same exact string!

File: refinery/core/models.py [hunk 2]
```diff
@@ -1740,6 +1744,26 @@ def attach_outputs_downloads(self):
...
+                file_store_item = FileStoreItem.objects.get(uuid=uuid)
+            except (FileStoreItem.DoesNotExist,
+                    FileStoreItem.MultipleObjectsReturned) as e:
+                logger.error(
+                    "Couldn't properly fetch FileStoreItem with UUID: %s %s",
+                    uuid,
+                    e
+                )
+            else:
+                terminate_file_import_task(file_store_item.import_task_id)
```
Comment (scottx611x) [inline refinery/core/models.py]: `terminate_file_import_task` can stay as a FileStoreitem model method

**Pull request merged**